pkg/monitor/monitor.go

```diff
@@ -25,8 +29,12 @@ type RuntimeConfig struct {
 }
 
 func Monitor(clusterName, clusterDomain, templatePath, cfgPath, apiVip string, apiPort, lbPort, statPort uint16, interval time.Duration) error {
-	var oldConfig, newConfig *config.ApiLBConfig
-	var k8sIsHealthy bool = false
+	var appliedConfig, curConfig, prevConfig *ApiLBConfig
```

Comment:
Go vet is failing on this. I'm assuming it needs to be *config.ApiLBConfig like it was before.

Reply:
Ohh,  I missed that on the rebase process - 10x!